new-frontend/src/components/v-summary-charts.vue

```diff
@@ -348,7 +348,7 @@ export default {
         zFromRamp: false,
         zFilterSearch: filterSearch,
       };
-      this.$emit('restoreZoomCommits', info);
+      this.$store.commit('updateTabZoomInfo', info);
```

Comment:
This one should better be done in PR #1390. Do notice that, we are only changing related code with the topic of this PR. Pls only add necessary changes in this PR.